docs-source/content/userguide/introduction/introduction.md

```diff
@@ -57,10 +75,12 @@ WebLogic Server and the WebLogic Kubernetes operator are certified and supported
 
 ### OpenShift
 
-Operator 2.0.1+ is certified for use on OpenShift Container Platform 3.11.43+, with Kubernetes 1.11.5+.  OpenShift 4 certification is currently in progress.
+Operator 2.0.1+ is certified for use on OpenShift Container Platform 3.11.43+, with Kubernetes 1.11.5+.  
+
+Operator 2.5.0+ is certified for use on OpenShift Container Platform 4.3.0+ with Kubernetes 1.16.2+.
```

Comment:
This has to be the wrong Kubernetes version.  Is it 1.13.2+?

Reply:
wow that is super interesting ins't it!!!!! 
![Screenshot from 2020-03-12 11-32-05](https://user-images.githubusercontent.com/6345393/76538129-3147ac80-6455-11ea-812f-bcaafd29ce6e.png)

Reply:
So, they have a different TLS implementation from ordinary Kubernetes? :)